packages/dds/ink/src/test/ink.spec.ts

```diff
@@ -109,65 +111,66 @@ describe("Ink", () => {
 			assert.equal(stroke.points.length, 1, "There should be only one point in the stroke");
 			assert.deepEqual(stroke.points[0], inkPoint, "The ink point is incorrect");
 		});
-	});
-
-	describe("Ink op processing in local state", () => {
-		it("should correctly process operations sent in local state", async () => {
-			// Set the data store runtime to local.
-			dataStoreRuntime.local = true;
-
-			// Create a stroke in local state.
-			const strokeId = ink.createStroke(pen).id;
-
-			// Load a new Ink in connected state from the snapshot of the first one.
-			const containerRuntimeFactory = new MockContainerRuntimeFactory();
-			const dataStoreRuntime2 = new MockFluidDataStoreRuntime();
-			containerRuntimeFactory.createContainerRuntime(dataStoreRuntime2);
-			const services2 = MockSharedObjectServices.createFromSummary(
-				ink.getAttachSummary().summary,
-			);
-			services2.deltaConnection = dataStoreRuntime2.createDeltaConnection();
-
-			const ink2 = new Ink(dataStoreRuntime2, "ink2", InkFactory.Attributes);
-			await ink2.load(services2);
 
-			// Now connect the first Ink
-			dataStoreRuntime.local = false;
-			containerRuntimeFactory.createContainerRuntime(dataStoreRuntime);
-			const services1 = {
-				deltaConnection: dataStoreRuntime.createDeltaConnection(),
-				objectStorage: new MockStorage(undefined),
-			};
-			ink.connect(services1);
-
-			// Verify that both the inks have the stroke.
-			assert.ok(ink.getStroke(strokeId), "The first ink does not have the stroke");
-			assert.ok(ink2.getStroke(strokeId), "The second ink does not have the stroke");
-
-			// Add a point to the stroke in the second ink.
-			const inkPoint = {
-				x: 10,
-				y: 10,
-				time: Date.now(),
-				pressure: 10,
-			};
-			ink2.appendPointToStroke(inkPoint, strokeId);
-
-			// Process the message.
-			containerRuntimeFactory.processAllMessages();
-
-			// Verify that both the inks have the added point.
-			const points1 = ink.getStroke(strokeId).points;
-			assert.equal(points1.length, 1, "There should be only one point in the stroke");
-			assert.deepEqual(points1[0], inkPoint, "The ink point is incorrect");
-
-			const points2 = ink2.getStroke(strokeId).points;
-			assert.equal(
-				points2.length,
-				1,
-				"There should be only one point in the stroke in remote client",
-			);
-			assert.deepEqual(points2[0], inkPoint, "The ink point is incorrect in remote client");
+		describe("Ink op processing in local state", () => {
+			it("should correctly process operations sent in local state", async () => {
```

Comment:
Looks like this describe block is missing the requisite beforeEach to create the ink as detached now that the `dataStoreRuntime.local = true;` is removed.  Probably just eliminate the beforeEach "createInk" and just let each describe block do its own setup since the runtimes and inks look different with this change.